repos/system_upgrade/el7toel8/actors/scheduleselinuxrelabeling/actor.py

```diff
@@ -23,17 +26,21 @@ def process(self):
                 try:
                     with open('/.autorelabel', 'w'):
                         pass
-                    report_generic(
-                        title='SElinux scheduled for relabelling',
-                        summary='/.autorelabel file touched on root in order to schedule SElinux relabelling.',
-                        severity='low',
-                    )
+                    create_report([
+                        reporting.Title('SElinux scheduled for relabelling'),
+                        reporting.Summary(
+                            '/.autorelabel file touched on root in order to schedule SElinux relabelling.'),
+                        reporting.Severity(reporting.Severity.LOW),
+                        reporting.Tags(COMMON_REPORT_TAGS),
+                    ])
+
                 except OSError as e:
                     # FIXME: add an "action required" flag later
-                    report_with_remediation(
-                        title='Could not schedule SElinux for relabelling',
-                        summary='./autorelabel file could not be created: {}.'.format(e),
-                        remediation='Please set autorelabelling manually after the upgrade.',
-                        severity='high'
-                    )
+                    create_report([
+                        reporting.Title('Could not schedule SElinux for relabelling'),
+                        reporting.Summary('./autorelabel file could not be created: {}.'.format(e)),
+                        reporting.Severity(reporting.Severity.HIGH),
+                        reporting.Tags(COMMON_REPORT_TAGS),
+                        reporting.Flags([reporting.Flags.FAILURE, reporting.Flags.INHIBITOR])
```

Comment:
isn't it too late for inhibitor at this stage?

Reply:
yep, did not realized that back then. I think I made the same brain-fart even during the first reporting phase :)